benchmarks/performance-poetry/complex-poetry/src/main/java/com/squareup/benchmarks/performance/complex/poetry/PerformancePoetryComponent.kt

```diff
@@ -1,12 +1,20 @@
 package com.squareup.benchmarks.performance.complex.poetry
 
+import android.annotation.SuppressLint
+import android.os.Trace
```

Comment:
Use the Android X trace instead. Also check out Trace.kt in POS to learn about some of the gotchas

Reply:
I thought I was using that - obviously the import messed up.